tools/qontract_cli.py

```diff
@@ -940,24 +948,19 @@ def user_credentials_migrate_output(ctx, account_name, output_path) -> None:
     )
     credentials = []
     for account, output in tf.outputs.items():
-        if account in skip_accounts:
```

Comment:
Why is `skip_accounts` removed here?

Reply:
Good catch! I initially copy and pasted that migration CLI from the get user credentials one. However, I did not remove this line. But it should be fine to delete, cause we are only migrating one account from the parameter (see setup in line 935). 
Further: my intention is to remove this code once the rollout is done.